libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFSmartStoreTests.m

```diff
@@ -690,7 +690,7 @@ - (void)removeStoreForManager:(SFSmartStoreDatabaseManager *)dbMgr withName:(NSS
 - (void)createDbDir:(NSString *)dbName withManager:(SFSmartStoreDatabaseManager *)dbMgr
 {
     NSError *createError = nil;
-    [dbMgr createStoreDir:dbName error:&createError];
+    [dbMgr createStoreDir:dbName];
     XCTAssertNil(createError, @"Error creating store dir: %@", [createError localizedDescription]);
```

Comment:
I think it would make more sense to check the return value (BOOL) of the method "createStoreDir" b/c createError is always going to be nil now (i.e. it's never modified).